babel/test/evaluate.js

```diff
@@ -1,72 +1,86 @@
 import test from 'ava';
 import * as _ from 'lodash';
-import {names} from './data/names';
-import {orbs} from './data/orbs';
-import {testMatchData} from './data/evaluate';
-import {nonMatchDropped} from './data/evaluate';
-import {droppedRandoms} from './data/evaluate';
-import {unaffectedOrbs} from './data/evaluate';
-import {evaluate} from '../src/board';
-import {findMatches} from '../src/board';
-import {combineMatches} from '../src/board';
 import {Board} from '../src/board';
-let board;
-let evaluatedOrbs;
-let matchDatas;
-let name;
+import {boards} from './data/boards';
+let boardInstance = new Board(5, 5);
 
-test.before(() => {
-    board = new Board(5, 5);
-    evaluatedOrbs = [];
-    matchDatas = [];
+/*
 
-    _.each(_.range(orbs.length), i => {
-        board.orbs = orbs[i];
-        let matchData = board.evaluate(combineMatches(findMatches(board.orbs)), [7, 8]);
-        evaluatedOrbs.push(board.orbs);
-        matchDatas.push(matchData);
-    });  
-});
+Collects data for each `board` after evaluation and stores it in `testBoards`.
+This data is then compared to the correct data in `boards` for the tests.
+
+Passing tests could look like this:
 
-_.each(_.range(orbs.length), j => {
-    name = names[j];
+testBoards -> {
+    'a simple row match': {
+        matchData: [
+            [6, 3]
+        ],
+        evaluatedOrbs: [
+            [ 1, 7, 8, 7, 5 ],
+            [ 5, 2, 3, 4, 4 ],
+            [ 4, 1, 2, 3, 3 ],
+            [ 3, 5, 1, 2, 2 ],
+            [ 2, 3, 4, 5, 1 ]
+        ]
+    },
+    'a simple column match': {
+        matchData: [
+            [6, 3]
+        ],
+        evaluatedOrbs: [
+            [ 1, 2, 3, 4, 7 ],
+            [ 5, 1, 2, 3, 8 ],
+            [ 4, 5, 1, 2, 7 ],
+            [ 3, 4, 5, 1, 5 ],
+            [ 2, 3, 4, 5, 4 ]
+        ]
+    },
+    // and so on...
+}
 
+*/
```

Comment:
I have looked at JSDoc, but I wasn't sure how to notate this since it isn't a function or method or anything. It is just compiling data to be tested.